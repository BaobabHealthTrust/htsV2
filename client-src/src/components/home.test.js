import React from 'react';
import ReactDOM from 'react-dom';
import Home from './home';
import renderer from 'react-test-renderer';
import toJson from 'enzyme-to-json';
import Enzyme, {
    shallow
} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

Enzyme.configure({
    adapter: new Adapter()
});

describe('Component: Home', () => {

    it('should match its empty snapshot', () => {
        const tree = renderer.create( 
            <Home />
        ).toJSON();

        expect(tree).toMatchSnapshot();
    })

})