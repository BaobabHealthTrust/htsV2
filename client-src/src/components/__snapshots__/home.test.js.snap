// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: Home should match its empty snapshot 1`] = `
<table
  border="0"
  style={
    Object {
      "borderCollapse": "collapse",
    }
  }
  width="100%"
>
  <tbody>
    <tr>
      <td
        style={
          Object {
            "width": "250px",
          }
        }
      >
        <div
          className="u3"
        >
          <table
            border="1"
            cellPadding="5"
            cellSpacing="0"
            style={
              Object {
                "border": "1px solid #345db5",
                "borderCollapse": "collapse",
                "width": "100%",
              }
            }
          >
            <tbody>
              <tr
                style={
                  Object {
                    "backgroundColor": "#345db5",
                  }
                }
              >
                <td
                  style={
                    Object {
                      "color": "#fff",
                      "fontSize": "12px",
                      "fontWeight": "bold",
                      "paddingLeft": "10px",
                    }
                  }
                >
                  Programs
                </td>
              </tr>
              <tr>
                <td>
                  <div
                    className="u3TabContent"
                  >
                    <ul
                      style={
                        Object {
                          "listStyle": "none",
                          "margin": "0px",
                          "padding": "0px",
                          "width": "100%",
                        }
                      }
                    />
                  </div>
                </td>
              </tr>
            </tbody>
          </table>
        </div>
      </td>
      <td
        style={
          Object {
            "verticalAlign": "top",
          }
        }
      >
        <div
          className="u4"
        >
          <table
            border="1"
            cellPadding="5"
            cellSpacing="0"
            style={
              Object {
                "border": "1px solid #345db5",
                "borderCollapse": "collapse",
                "width": "100%",
              }
            }
          >
            <tbody>
              <tr
                style={
                  Object {
                    "backgroundColor": "#345db5",
                  }
                }
              >
                <td
                  style={
                    Object {
                      "color": "#fff",
                      "fontSize": "12px",
                      "fontWeight": "bold",
                      "paddingLeft": "10px",
                      "textAlign": "center",
                    }
                  }
                >
                  ...
                </td>
              </tr>
              <tr>
                <td
                  style={
                    Object {
                      "padding": 0,
                    }
                  }
                >
                  <div
                    className="u4TabContent"
                  />
                </td>
              </tr>
            </tbody>
          </table>
        </div>
      </td>
      <td
        style={
          Object {
            "width": "200px",
          }
        }
      >
        <div
          className="u5"
        >
          <table
            border="1"
            cellPadding="5"
            cellSpacing="0"
            style={
              Object {
                "border": "1px solid #345db5",
                "borderCollapse": "collapse",
                "width": "100%",
              }
            }
          >
            <tbody>
              <tr
                style={
                  Object {
                    "backgroundColor": "#345db5",
                  }
                }
              >
                <td
                  style={
                    Object {
                      "color": "#fff",
                      "fontSize": "12px",
                      "fontWeight": "bold",
                      "paddingLeft": "10px",
                    }
                  }
                >
                  Menu
                </td>
              </tr>
              <tr>
                <td
                  style={
                    Object {
                      "padding": 0,
                    }
                  }
                >
                  <div
                    className="u5TabContent"
                  >
                    <ul
                      style={
                        Object {
                          "listStyle": "none",
                          "margin": "0px",
                          "padding": "0px",
                          "width": "100%",
                        }
                      }
                    />
                  </div>
                </td>
              </tr>
            </tbody>
          </table>
        </div>
      </td>
    </tr>
  </tbody>
</table>
`;
